test-utils/testlib/src/runtime_utils.rs

```diff
@@ -23,28 +21,27 @@ pub fn x_dot_y_dot_alice_account() -> AccountId {
     "x.y.alice.near".parse().unwrap()
 }
 
-static DEFAULT_TEST_CONTRACT_HASH: Lazy<CryptoHash> =
-    Lazy::new(|| hash(near_test_contracts::rs_contract()));
-
 pub fn add_test_contract(genesis: &mut Genesis, account_id: &AccountId) {
+    add_contract(genesis, account_id, near_test_contracts::rs_contract().to_vec())
```

Comment:
also comment this function (add_test_contract) -- which contract does it add ?

Reply:
done